package.json

```
@@ -65,15 +65,17 @@
     "reselect": "4.0.0",
     "semver": "6.3.0",
     "superagent": "5.1.0",
-    "typescript": "3.6.4",
+    "typescript": "3.7.4",
     "xregexp": "4.2.4",
     "zen-observable": "0.8.14"
   },
   "devDependencies": {
     "@babel/cli": "7.6.4",
     "@babel/core": "7.6.4",
     "@babel/plugin-proposal-class-properties": "7.5.5",
+    "@babel/plugin-proposal-nullish-coalescing-operator": "^7.7.4",
     "@babel/plugin-proposal-object-rest-spread": "7.6.2",
+    "@babel/plugin-proposal-optional-chaining": "^7.7.5",
```

Please use fix versions
```suggestion
    "@babel/plugin-proposal-optional-chaining": "7.7.5",
```

---

Thanks for catching that! And for mentioning the codemod—I wasn't aware of that one.